apps/angular/pipe-hard/src/app/util-func.pipe.ts

```diff
@@ -0,0 +1,14 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+  name: 'utilFunc',
+  standalone: true,
+})
+export class UtilFuncPipe implements PipeTransform {
```

Comment:
This is really good as a generic pipe, but it doesn't fit the requirements:
`The goal is to create a specific pipe for this utils file, where you will need to pass the name of the function you want to call and the needed arguments.`